CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased]
 
+## Changed
+* Trivia not restored after long identifier in SynField. [#2290](https://github.com/fsprojects/fantomas/pull/2290)
```

Comment:
Sorry to be picky here, but it makes more sense to put this under `Fixed`.
Changed would more be applicable if the style is changed.